requirements_test.txt

```diff
@@ -13,5 +13,6 @@ imageio-ffmpeg
 colorcet
 cmocean
 scooby>=0.5.1
-meshio>=3.3.0
+meshio>=4.0.1
+h5py
```

Comment:
Why would we require `h5py` for testing? We didn't need it before.

Reply:
Seems that `test_meshio.py` fails without h5py, and that `h5py` isn't a required dependency of `meshio`.  I'm find with adding it here.

Reply:
We don't need it, but `meshio` 4+ does.